src/newusers.c

```diff
@@ -1244,21 +1229,22 @@ int main (int argc, char **argv)
 				fprintf(stderr,
 					_("%s: line %d: homedir must be an absolute path\n"),
 					Prog, line);
-				errors++;
-				continue;
-			};
+				fail_exit (EXIT_FAILURE);
+			}
 			if (mkdir (newpw.pw_dir, mode) != 0) {
 				fprintf (stderr,
 				         _("%s: line %d: mkdir %s failed: %s\n"),
 				         Prog, line, newpw.pw_dir,
 				         strerror (errno));
-			} else if (chown (newpw.pw_dir,
-			                  newpw.pw_uid,
-			                  newpw.pw_gid) != 0) {
+				fail_exit (EXIT_FAILURE);
```

Comment:
This previously did not raise an error.  I assume that was so that an admin could pre-create a homedir for a new user.  That will no longer be possible.  Should we simply log here, but continue?

Reply:
or, check for errno == EXIST and fail otherwise?

Reply:
Checking `EEXIST` makes sense to me.

Reply:
In case of EEXST, should we log, or just do nothing?  I think I'll keep the log, just for keeping old behavior and not surprising users.